dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -0,0 +1,47 @@
+# == Schema Information
+#
+# Table name: plc_enrollment_unit_assignments
+#
+#  id                            :integer          not null, primary key
+#  plc_user_course_enrollment_id :integer
+#  plc_course_unit_id            :integer
+#  status                        :string(255)
+#  created_at                    :datetime         not null
+#  updated_at                    :datetime         not null
+#
+# Indexes
+#
+#  enrollment_unit_assignment_course_enrollment_index  (plc_user_course_enrollment_id)
+#  enrollment_unit_assignment_course_unit_index        (plc_course_unit_id)
+#
+
+class Plc::EnrollmentUnitAssignment < ActiveRecord::Base
+  belongs_to :plc_user_course_enrollment, class_name: '::Plc::UserCourseEnrollment'
+  belongs_to :plc_course_unit, class_name: '::Plc::CourseUnit'
+
+  has_many :plc_module_assignments, class_name: '::Plc::EnrollmentModuleAssignment', foreign_key: 'plc_enrollment_unit_assignment_id', dependent: :destroy
+  has_many :plc_task_assignments, through: :plc_module_assignments, class_name: '::Plc::EnrollmentTaskAssignment', dependent: :destroy
+
```

Comment:
This logic came over from user_course_enrollment. How users get their units assigned has not been built here (didn't want the PR to sprawl)- for testing purposes, I'm just going to do it manually until that gets built out.